.zenodo.json

```diff
@@ -0,0 +1,49 @@
+{
+  "title": "Downscaling CMIP6 with multiple statistical methods",
+  "creators": [
+    {
+      "name": "Oriana S Chegwidden",
+      "affiliation": "CarbonPlan",
+      "orcid": "0000-0003-1376-3835"
+    },
+    {
+      "name": "Raphael Hagen",
+      "affiliation": "CarbonPlan"
+    },
+    {
+      "name": "Kata Martin",
+      "affiliation": "CarbonPlan"
+    },
+    {
+      "name": "Max Jones",
+      "affiliation": "CarbonPlan",
+      "orcid": "0000-0003-0180-8928"
+    },
+    {
+      "name": "Anderson Banihirwe",
+      "affiliation": "CarbonPlan",
+      "orcid": "0000-0001-6583-571X"
+    },
+    {
+      "name": "Cindy Chiao"
+    },
+    {
+      "name": "Sadie Frank",
+      "affiliation": "CarbonPlan"
+    },
+    {
+      "name": "Jeremy Freeman",
+      "affiliation": "CarbonPlan",
+      "orcid": "0000-0001-7077-7972"
+    },
+    {
+      "name": "Joe Hamman",
+      "affiliation": "CarbonPlan",
+      "orcid": "0000-0001-7479-8439"
+    }
+  ],
+  "upload_type": "software",
+  "keywords": ["downscaling", "CMIP6", "bias-correction", "climate"],
+  "grants": [{ "id": "Microsoft AI for Earth" }],
+  "description": "The codebase here was used by [CarbonPlan](https://carbonplan.org) to develop a collection of datasets featuring climate projections from the Coupled Model Intercomparison Project Phase 6, bias-corrected and downscaled to the 0.25 degree resolution. The datasets include daily maximum and minimum temperatures and daily precipitation spanning the entire globe over the time period 1950-2100. Notably, the dataset collection includes results from multiple downscaling methods, allowing users to better evaluate the effects of downscaling for their specific application of interest. More details on the project are available in CarbonPlan’s [explainer article](https://carbonplan.org/research/cmip6-downscaling-explainer). Also, you can interactively explore the data through a [web tool](https://carbonplan.org/research/cmip6-downscaling) and access the data via an [example Jupyter notebook](https://github.com/carbonplan/cmip6-downscaling/blob/main/notebooks/accessing_data_example.ipynb)."
```

Comment:
@orianac, i'm not very confident about whether the markdown gets rendered. i've seen some version of this JSON file with HTML in it though (e.g. https://github.com/tensorflow/tensorflow/blob/master/.zenodo.json) --> https://zenodo.org/record/6574269

Reply:
more specifically the `notes` section: https://github.com/tensorflow/tensorflow/blob/360fe7abad4d8f0f3e286f15f9b96257a8a2b90c/.zenodo.json?rgh-link-date=2022-10-04T21%3A46%3A41Z#L12

Reply:
Perhaps we should convert the markdown to HTML before merging??

Reply:
sounds great- will do right now